atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -650,13 +660,158 @@ def set_build_image(self):
             raise RuntimeError("Build object is malformed, failed to fetch buildroot image")
 
         if build_kind == 'DockerImage':
-            self.config_kwargs['build_image'] = build_name
+            return build_name
         else:
             raise RuntimeError("Build kind isn't 'DockerImage' but %s" % build_kind)
 
+    def set_build_image(self, platforms):
+        """
+        Overrides build_image for worker, to be same as in orchestrator build
+        """
+        orchestrator_plat = 'x86_64'
+        status = get_build_json().get("status")
+        build_config_name = None
+        build_image = None
+        imagestream = None
+        need_manifest = not (orchestrator_plat in platforms and len(platforms) == 1)
+        use_same_image = False
+        current_buildimage = self.get_current_buildimage()
+        self.plat_digests = {}
```

Comment:
`platform_digests`, and should we initialize this in `__init__`?

Reply:
sure